test/unit/commands/network-specs.js

```diff
@@ -145,8 +145,16 @@ describe('Network', function () {
   describe('setGeoLocation', function () {
     it('should set location', async function () {
```

Comment:
This is really not test that it "should set location" so much as that it returns the geo location.

Reply:
which name would be better?

Reply:
Something like?
```js
it('should return location in use after setting', async function () {
```

Reply:
fixed